src/commands/run.ts

```diff
@@ -41,11 +44,17 @@ async run() {
   // It would be desirable to execute each iteration of CI in a cycle
   if (flags.iterations) {
     for (let i = 0; i < flags.iterations; i++) {
+      console.log("running for "+i+1+"th iteration")
       if (flags.uptime)
         queue.add(update)
       if (flags.responseTime)
         queue.add(() => update(true))
       if (flags.graphs)
+        // method to call data point function
```

Comment:
Hard-coded values

Reply:
removed them, this was just for reference.